dist/components/progress-steps/_progress-steps.scss

```diff
@@ -0,0 +1,108 @@
+// Progress
+// ========
+//
+// Helps show users where they exist within a multiple-page workflow, such as a
+// checkout flow.
+
+// Configurable Variables
+// ---
+
+$progress-steps__item-bg-color: hsl(0, 0%, 95%) !default;
+$progress-steps__item-padding: if(variable-exists(v-space), $v-space, 10px) !default;
+$progress-steps__layer: if(variable-exists(stage-layer), $stage-layer, 10) !default;
+$progress-steps__current-bg-color: darken($progress-steps__item-bg-color, 10%) !default;
+$progress-steps__completed-bg-color: darken($progress-steps__item-bg-color, 20%) !default;
+
+// Arrow Divider Variables
+// 1. These two variables together determine the angle of the arrow dividers
+
+$progress-steps__divider-angle: 90px !default; // 1
+$progress-steps__divider-width: 20px !default; // 1
+$progress-steps__divider-border-thickness: 2px !default;
+$progress-steps__divider-border-color: hsl(360, 100%, 100%) !default;
+
+
+.c-progress-steps {
+    display: flex;
+    overflow: hidden;
+}
+
+
+// Progress: Steps
+// ---
+
+.c-progress-steps__item {
+    position: relative; // For the absolutely positioned arrows
+
+    display: block;
+    padding: $progress-steps__item-padding;
+
+    background-color: $progress-steps__item-bg-color;
+
+    // Adjust padding to account for the arrows
+    &:not(:first-child) {
+        padding-right: 0;
+        padding-left: $progress-steps__divider-width + $progress-steps__item-padding;
+    }
+
+    // Arrows
+    &::after,
+    &::before {
+        content: '';
+
+        position: absolute;
+        top: 50%;
+        left: 100%;
+        z-index: $progress-steps__layer;
+
+        display: block;
+        display: block;
+        width: 0;
+        height: 0;
+        border-top: $progress-steps__divider-angle solid transparent;
+        border-bottom: $progress-steps__divider-angle solid transparent;
+
+        transform: translateY(-50%);
+    }
+
+    // coloured triangle
+    &::after {
+        border-left: $progress-steps__divider-width solid $progress-steps__item-bg-color;
+    }
+
+    // background color spacing
+    &::before {
+        margin-left: $progress-steps__divider-border-thickness; // offset to create the appearance of a border
+        border-left: $progress-steps__divider-width solid $progress-steps__divider-border-color;
+    }
+}
+
+.c-progress-steps__item.c--current {
+    background-color: $progress-steps__current-bg-color;
+
+    &::after {
+        border-left-color: $progress-steps__current-bg-color;
+    }
+}
+
+.c-progress-steps__item.c--completed {
+    background-color: $progress-steps__completed-bg-color;
+
+    &::after {
+        border-left-color: $progress-steps__completed-bg-color;
+    }
+}
+
+
+// Progress: Index
+// ---
+
+.c-progress-steps__index {
```

Comment:
Is there a need for these? I suppose they're here just to demonstrate that they are available for customization?

Perhaps some comments to document that would be in order.